content/news/023/index.md

```diff
@@ -180,6 +180,40 @@ outline shader in the Bevy render pipeline.
 [themengi-twitter]: https://twitter.com/voooooogel
 [themengi-video]: https://youtube.com/watch?v=gtIphiK7tMs
 
+### [Veloren][veloren]
+
+![Cave run](veloren.png) _Turns out the cave wasn't empty_
+
+[Veloren][veloren] is an open world, open-source voxel RPG inspired by Dwarf
+Fortress and Cube World.
+
+In June, Veloren released 0.10. During the release party, some of the developers
+joined a voice call to stream the game and answer some questions about
+development. You can watch that stream [here][veloren-stream]. A lot of metrics
+were recorded during the launch, and you can read about them in [blog
+#124][veloren-124]. At peak, 118 players were playing at the same time during
+the launch.
```

Comment:
Linter is getting a bit confused by this link being on two lines:

```suggestion
were recorded during the launch, and you can read about them in
[blog #124][veloren-124]. At peak, 118 players were playing at the same
time during the launch.
```

Reply:
Good catch, thanks :)